### PR TITLE
[TRANSLATION] EN to JA: your-first-progressive-web-app/index.markdown

### PR DESCRIPTION
**Pull request by yoichiro**
I translated the page `your-first-progressive-web-app/index.markdown` from en to ja. If this pull request looks good to the reviewer, it would be great if this pull request is merged to the master branch. Thank you for his/her confirmation.

### CODE REVIEW

**Comment by yoichiro** [comment]
I signed it!

**Comment by PaulKinlan** [comment]
@agektmr I preseume we are all ok with this :)

File: .gitignore [hunk 1]
```diff
@@ -19,3 +19,4 @@ appengine/build
 tools/shows-gen/yt-api-key.json
 src/jekyll/_includes/svgs/
 .bundle/
+google_appengine/
```
Comment (agektmr) [inline .gitignore]: @PaulKinlan @petele Just in case this won't surprise you. Basically let git ignore AppEngine directory. I think this is relevant.

**Comment by agektmr** [comment]
Translation LGTM.
@yoichiro Can you add yourself to [_contributors.yaml](https://github.com/google/WebFundamentals/blob/master/src/content/_contributors.yaml)?

File: .gitignore (hunk 1 above)
Comment (yoichiro) [inline .gitignore]: Yes, the README.md doc instructs contributors to extract the AppEngine SDK into the project root directory. Therefore, I think that we should avoid that the SDK directory is added to git repository with the .gitignore file.

**Comment by yoichiro** [comment]
@agektmr Ok, I just added me by myself as a contributor into the file on 81ba06d.

**Comment by yoichiro** [comment]
@PaulKinlan Thanks, Paul!

**Pull request merged**